packages/styled-components/src/hoc/withTheme.js

```diff
@@ -1,11 +1,9 @@
 // @flow
 import React, { type ComponentType } from 'react';
-import { ThemeConsumer, type Theme } from '../models/ThemeProvider';
-import determineTheme from '../utils/determineTheme';
-import getComponentName from '../utils/getComponentName';
-import hoistStatics from '../utils/hoist';
```

Comment:
this is an example of the default export is bad. the name of the function is misleading since it is hoist non statics... but you can give any name since it is the default export...

Reply:
I don’t really understand the beef with default exports. The point is the implementation is opaque and the end developer doesn’t need to know how the internals are structured to use the thing.